src/mbgl/gl/debugging.cpp

```diff
@@ -190,5 +189,3 @@ group::~group() {
 } // namespace debugging
 } // namespace gl
 } // namespace mbgl
-
-#endif
```

Comment:
Fixed! (We do still try to load the `GL_KHR_debug` extension, but that's the intent of this PR.)